luda-editor/luda-editor-client/src/app/editor/job/change_image.rs

```diff
@@ -0,0 +1,25 @@
+use super::JobExecute;
+use crate::app::types::*;
+
+#[derive(Debug, Clone)]
+pub struct ChangeImageJob {
+    pub clip_id: String,
+    pub character_pose_emotion: CharacterPoseEmotion,
+}
+
+impl JobExecute for ChangeImageJob {
+    fn execute(&self, sequence: &Sequence) -> Result<Sequence, String> {
+        let sequence = sequence.clone();
+        match sequence.replace_clip(&self.clip_id, |clip: &CameraClip| {
+            let mut clip = clip.clone();
+            if clip.camera_angle.character_pose_emotion != self.character_pose_emotion {
+                clip.camera_angle.character_pose_emotion = self.character_pose_emotion.clone();
+            }
+            Ok(clip)
+        }) {
+            UpdateResult::Updated(replacer) => Ok(replacer),
+            UpdateResult::NotUpdated => Err("Subtitle clip not found".to_string()),
```

Comment:
Is it right to return `Subtitle clip not found` error?

Reply:
Yes, it should find the clip.
Error message is wrong. it should be Camera clip. i will change it.

Reply:
Fixed

Reply:
super fast